components/fxa-client/src/internal/device.rs

```diff
@@ -7,14 +7,15 @@ use std::collections::{HashMap, HashSet};
 use serde_derive::*;
 
 pub use super::http_client::{
-    DeviceLocation as Location, DeviceType as Type, GetDeviceResponse as Device, PushSubscription,
+    DeviceLocation as Location, GetDeviceResponse as Device, PushSubscription,
 };
 use super::{
     commands::{self, IncomingDeviceCommand},
     error::*,
     http_client::{DeviceUpdateRequest, DeviceUpdateRequestBuilder, PendingCommand},
     telemetry, util, CachedResponse, FirefoxAccount,
 };
+use crate::DeviceType;
```

Comment:
Why `crate::DeviceType` over `sync15::DeviceType`?  It feels slightly non-standard to me, but I can't see any issues either.

Reply:
Because I was lazy :) I've fixed this.